templates/bridge-whatsapp/deployment.yaml

```diff
@@ -56,17 +56,17 @@ spec:
           image: "{{ .Values.bridges.whatsapp.image.repository }}:{{ .Values.bridges.whatsapp.image.tag }}"
           imagePullPolicy: {{ .Values.bridges.whatsapp.image.pullPolicy }}
           command: ["sh"]
-          args: ["-c", "cp /load/config.yaml /data/config.yaml"]
+          args: ["-c", "cp /load/config.yaml /data/config.yaml; touch /bridges/whatsapp.yaml; chown 1000:1000 /data/config.yaml /bridges/whatsapp.yaml"]
           volumeMounts:
             - name: data
               mountPath: /data
             - name: config
               mountPath: /load
               readOnly: true
+            - name: bridges
+              mountPath: /bridges
           securityContext:
-            capabilities:
-              drop:
-                - ALL
+            runAsUser: 0
```

Comment:
I've tried very hard to run as many containers rootless as possible in this chart. I don't see why this container should need to manually fiddle with file permissions. What's the context behind this change? And why is it only needed for the WhatsApp bridge?

Reply:
> I've tried very hard to run as many containers rootless as possible in this chart.
I'm really appreciate that..  Sorry, this Is related with #46... On my primary Environment (Ubunut CDK over Openstack and Cinder)  get:
![image](https://user-images.githubusercontent.com/824129/95597372-25848e80-0a14-11eb-9489-eafa3a32c1c4.png)
`$ kubectl logs pod/matrixtest-bridge-whatsapp-6d7b9c87-77lsl -c generate-config
Failed to save registration: open /bridges/whatsapp.yaml: permission denied
`